R/misc.R

```diff
@@ -302,104 +309,4 @@ load_regional_multivariate_data <- function(matrix_y_min_complete = NULL, # when
         chrom = dat$chrom,
         grange = dat$grange
         ))
-}
```

Comment:
@Travyse this is some code I wrote today but you overwrite and deleted it. Please recover it. Before you send in PR you should sycn your branch first. Otherwise you'll overwrite other peoples' updates.